### DISCUSSION TITLE
Version 3 Prep

### DISCUSSION THREAD

**Discussion by jenniferyee** [rpoleski/MulensModel#102] [Ideas]
In preparation for Version 3, I suggest we save png copies of all the use_case/example plots, so we know the expected output. Vote up or down.

**Comment by jenniferyee** [comment]
- [ ]  save png copies of all the use_case/example plot
- [ ] consider whether implementing astrometric information/fitting will affect the architecture update
- [ ] articulate goals for version 3

**Comment by jenniferyee** [comment]
@rpoleski Can you add the figures and output from example_07 (MultiNest)? Apparently, I still don't have it installed on my laptop.

**Comment by jenniferyee** [comment]
Examples that still need output saved:
- [ ] Ex07
- [ ] Ex08
- [ ] Ex16
- [x] Ex20
- [x] Ex21
- [x] Ex22
- [x] Ex23
- [x] Ex24

**Comment by rpoleski** [comment]
Ex07 is not producing any plots.

**Comment by rpoleski** [comment]
Closing. See #116
